automation/WinFormsAutomationApp/AuthenticationHelper.cs

```diff
@@ -118,7 +118,7 @@ internal static class AuthenticationHelper
                     {
                         var updated = item;
                         updated.Value.Result.ExpiresOn = DateTime.UtcNow;
-                        UpdateCache(item, updated);
+                        await UpdateCache(item, updated);
```

Comment:
all await calls should have ConfigureAwait(false);

Reply:
Ah. My bad. New commit is on the way!